pmg/models/resources.py

```diff
@@ -1297,12 +1297,19 @@ def meetings_by_member(cls):
         return rows
 
     @classmethod
-    def annual_attendance_trends(cls, from_year, to_year):
+    def annual_attendance_trends(cls,
+                                 from_year=None,
```

Comment:
from_year and to_year aren't used any more, start_date and end_date are hardcoded - this will be very confusing